lib/src/config/forgot_password_spacer_options.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+@immutable
+class ForgotPasswordSpacerOptions {
+  const ForgotPasswordSpacerOptions({
```

Comment:
where are the /// comments for the constructor and the ForgotPasswordSpacerOptions class?